ansible/roles/etcd/tasks/iptables.yml

```diff
@@ -24,3 +24,4 @@
   with_items:
     - "{{ etcd_client_legacy_port }}"
     - "{{ etcd_peer_legacy_port }}"
+  when: not is_atomic
```

Comment:
This should not be added. I think it's a workaround, instead of fixing the core issue. The project supports multiple networking implementations. The legacy ports are required (for the time being) by contiv networking.